site/_docs/themes.md

```diff
@@ -14,7 +14,11 @@ Jekyll has an extensive theme system, which allows you to leverage community-mai
 
 2. Save the changes to your `Gemfile`
 3. Run the command `bundle install` to install the theme
-4. Finally, activate the theme by adding the following to your site's `_config.yml`:
+4. Set your `/css/main.scss` to import your theme partials using the `@import` directive:
```

Comment:
Should we create the design pattern of using `/assets/style.scss? That way, when themes can support other files, we can just copy the assets folder (with images, js, stylesheets, etc.). Additionally, given this design pattern, I'm not sure how many sites will have multiple CSS files (maybe sass, but that'd be in `_sass`) to make the dedicated folder necessary.